src/commands/issuance/create-config.ts

```diff
@@ -0,0 +1,179 @@
+import z from 'zod'
+import { BaseCommand } from '../../common/base-command.js'
+import { DidMethods } from '../../common/constants.js'
+import { promptRequiredParameters } from '../../common/prompts.js'
+import { INPUT_LIMIT, validateInputLength } from '../../common/validators.js'
+import { issuanceService } from '../../services/affinidi/cis/service.js'
+import { cweService } from '../../services/affinidi/cwe/service.js'
+
+export class CreateIssuanceConfig extends BaseCommand<typeof CreateIssuanceConfig> {
+  static summary = 'Creates credential issuance configuration in your active project'
+  static examples = [
+    '<%= config.bin %> <%= command.id %> -n <value> -w <value> -f credentialSchemas.json',
+    '<%= config.bin %> <%= command.id %> --name <value> --wallet-id <value> --description <value> --credential-offer-duration <value> --file credentialSchemas.json',
+  ]
+  static flags = {
+    name: Flags.string({
+      char: 'n',
+      summary: 'Name of the credential issuance configuration',
+    }),
+    description: Flags.string({
+      char: 'd',
+      summary: 'Description of the credential issuance configuration',
+    }),
+    'wallet-id': Flags.string({
+      char: 'w',
+      summary: 'ID of the wallet',
+    }),
+    'credential-offer-duration': Flags.integer({
+      summary: 'Credential offer duration in seconds',
+    }),
+    file: Flags.string({
+      char: 'f',
+      summary:
+        'Location of a json file containing the list of allowed schemas for creating a credential offer. One or more schemas can be added to the issuance. The credential type ID must be unique',
+    }),
+  }
+
+  public async run(): Promise<IssuanceConfigDto> {
+    const { flags } = await this.parse(CreateIssuanceConfig)
+
+    const promptFlags = await promptRequiredParameters(['wallet-id', 'name', 'file'], flags)
+
+    const flagsSchema = z.object({
+      'wallet-id': z.string().max(INPUT_LIMIT),
+      name: z.string().min(3).max(INPUT_LIMIT),
+      description: z.string().max(INPUT_LIMIT).optional(),
+      'credential-offer-duration': z.number().optional(),
+      file: z.string(),
+    })
+    const validatedFlags = flagsSchema.parse(promptFlags)
+
+    ux.action.start('Fetching wallets')
+    const { wallets } = await cweService.listWallets()
+    ux.action.stop('Fetched successfully!')
+
+    let issuerWalletId = validatedFlags['wallet-id']
+
+    const walletIds = wallets?.map((wallet: WalletDto) => wallet.id) || []
+    const wrongAriProvided = !walletIds.includes(issuerWalletId)
+
+    if (!issuerWalletId || wallets?.length === 0 || wrongAriProvided) {
+      const walletChoices =
+        wallets?.map((wallet: WalletDto) => ({
+          name: wallet.name || wallet.id,
+          value: wallet.id,
+        })) || []
+
+      const CREATE_NEW_WALLET = 'Create new wallet'
+      walletChoices.push({ name: CREATE_NEW_WALLET, value: CREATE_NEW_WALLET })
+
+      const selectedWallet = await select({
```

Comment:
Check no-input flag before prompting

Reply:
The check happens [here](https://github.com/affinidi/affinidi-cli/pull/400/files/c3f412cdd6ee3768e5eda7dcca3a402a3dfe1fc7#diff-3f20c6f3e1210b31064248fb4e17f262614fbceb146be1be39ff33bc1485d58bR51) - and error will be thrown if no value provided

Reply:
<img width="1180" alt="Screenshot 2024-09-02 at 20 09 50" src="https://github.com/user-attachments/assets/12d5d25e-70e1-45f9-9f6b-9cbc2225b126">